docroot/modules/custom/mass_alerts/mass_alerts.module

```diff
@@ -40,26 +40,36 @@ function mass_alerts_get_scope(NodeInterface $node) {
  * Implements hook_form_FORM_ID_alter() for the FORM_ID() form.
  */
 function mass_alerts_form_node_alert_form_alter(&$form, FormStateInterface $form_state, $form_id) {
-  $form['#validate'][] = '_mass_alerts_validate_site_wide_alert_creation';
   $form['#validate'][] = 'mass_alerts_validate_alert_placement';
   $form['#validate'][] = 'mass_alerts_validate_alert_content';
 
-  if (!\Drupal::currentUser()->hasPermission('create site wide alerts')) {
-    unset($form['field_alert_display']['widget']['#options']['site_wide']);
-  }
+  // @TODO Remove the code, since we are using new Sitewide Alerts content type.
```

Comment:
Yeah, this is the deployment problem, since we need to migrate data from the field before deleting that option. SO the best and safest way is to delete it after deploy.